scripts/imgr

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+usage()
+{
+cat << EOF
+Usage: $0 IOCNAME [--hutch HUTCH] --COMMAND
+
+Script to control an ioc. 
+
+List of commands:
+$0 IOCNAME [--hutch HUTCH] --reboot soft
+$0 IOCNAME [--hutch HUTCH] --reboot hard
+$0 IOCNAME [--hutch HUTCH] --enable
+$0 IOCNAME [--hutch HUTCH] --disable
+$0 IOCNAME [--hutch HUTCH] --upgrade RELEASE_DIR
+$0 IOCNAME [--hutch HUTCH] --move HOST
+$0 IOCNAME [--hutch HUTCH] --move HOST:PORT
+$0 [--hutch HUTCH] --list [--host HOST] [--enabled_only|--disabled_only]
+ 
+EOF
+}
+
+if [ $# -lt 1 ]; then
+   echo 'need arguments' >&2
+   usage
+   exit 0
+fi
+
+
+if [[ ($1 == "--help") || ($1 == "-h") ]]; then
+   usage
+   exit 0
+fi
+
+ulimit -c unlimited 
+
+source /reg/g/pcds/setup/pathmunge.sh
+ldpathmunge /usr/local/lib
+
+DIRTMP=`dirname  "${BASH_SOURCE[0]}"`
+DIR="$( cd $DIRTMP && pwd -P )"
```

Comment:
No I don't think it does since the command is run in a subshell. It's still probably cleaner to use `DIR=$(readlink -f $(dirname  "${BASH_SOURCE[0]}")) though`